examples/ws_tutorial/ws_tutorial/app.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
I think we should remove this for now, this tutorial app is not typed at all anyway.

Reply:
it has some types, for example:
`async def process_request_ws(self, req: Request, ws: WebSocket):`

Reply:
Aha, checked quickly and didn't see many.
Regardless, as a user, why would I need this? I think it is useful for our own codebase, but users can use the standard mode too if they wish so?

Reply:
I'll add a skip in the file then